grpc/src/main/protobuf/blocks_api.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package com.wavesplatform.api.grpc;
+
+import "block.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
+
+service BlocksApi {
+    rpc GetBlock (BlockRequest) returns (BlockAndHeight);
+    rpc GetBlocksRange (BlocksRangeRequest) returns (stream BlockAndHeight);
+    rpc CalcBlocksDelay (BlocksDelayRequest) returns (google.protobuf.UInt64Value);
```

Comment:
How about we just drop this method? Block delays can easily be calculated from block headers.